### DISCUSSION TITLE
Advanced Settings would benefit from having sorting feature for its columns

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#134] [Ideas]
I propose adding of an alphabetical sorting feature that could be applied to

**_Advanced Options > Advanced Settings > Key_**

or even every remaining of the columns [i.e. _**Type**_ and _**Value**_]

Alternatively the minimum change that could be implemented and by doing so benefit users would be to leave the functionality of the _**Advanced Settings**_ as they are now [i.e. with fixed positions in the table], but just list the Key entries in an alphabetical way

**Comment by jonaskohl** [comment]
I will consider this for a future update
